templates/pages/helpdesk/index.html.twig

```diff
@@ -86,7 +86,7 @@
                         <div class="col-12 col-sm-6 col-md-4 d-flex">
                             <a
                                 class="card mx-1 my-2 flex-grow-1"
-                                href="{{ path(tile.getLink()) }}"
+                                href="{{ path(tile.getTileLink()) }}"
```

Comment:
This will not work for `ExternalPageTile`. Indeed, the `path()` function will prefix it with `$CFG_GLPI['root_doc']`. You have to change your `getTileLink()` to return a complete URL.
```suggestion
                                href="{{ tile.getTileLink() }}"
```

You should probably rename it to `getTileURL`.